pkg/lint/lintersdb/validator.go

```diff
@@ -17,19 +17,19 @@ func NewValidator(m *Manager) *Validator {
 	}
 }
 
-func (v Validator) validateLintersNames(cfg *config.Linters) error {
+func (v *Validator) validateLinterConfigNames(cfg *config.Linters) error {
```

Comment:
just curious why the method got changed to `validateLinterConfigNames` ? 

AFAIK this validation is applicable for `golangci-lint run -E 121231` as well.

Reply:
ok, fixed.